src/include/optimizer/stats/selectivity.h

```diff
@@ -0,0 +1,95 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// selectivity.h
+//
+// Identification: src/include/optimizer/stats/selectivity.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <vector>
+#include <algorithm>
+#include <memory>
+
+#include "type/value.h"
+#include "common/macros.h"
+#include "common/logger.h"
+#include "storage/data_table.h"
+#include "optimizer/stats/stats_storage.h"
+#include "optimizer/stats/tuple_samples_storage.h"
+#include "catalog/column_catalog.h"
+#include "catalog/catalog.h"
+#include "optimizer/stats/table_stats.h"
+#include "optimizer/stats/value_condition.h"
+#include "optimizer/stats/column_stats.h"
+
+namespace peloton {
+namespace optimizer {
+
+static constexpr double DEFAULT_SELECTIVITY = 0.5;
```

Comment:
@rohit-cmu To make the `static constexpr` compile (It doesn't compile when defined inside of the class)